code/swb_shared/Animations.cs

```diff
@@ -0,0 +1,13 @@
+
+namespace SWB.Shared;
+
+/// <summary>
+/// Animations that SWB may trigger on the player object
+/// </summary>
+public enum Animations
```

Comment:
To make this more extendable I would take the TagsHelper approach, create a partial class that can be extended that allows users to add their own animation string values. I'm not a big fan of restricting this, then we could also get rid of the enum => string mapping in the TriggerAnimation method.